include/wil/registry.h

```diff
@@ -26,3244 +26,3279 @@
-        //     WCHAR string_value[100]{};
-        //     uint32_t requiredBytes{};
-        //     hr = wil::reg::get_value_string_nothrow(key, L"string_value_name", string_value, &requiredBytes);
-        //
-        // Example of usage writing a REG_MULTI_SZ
-        //     wil::unique_cotaskmem_array_ptr<wil::unique_cotaskmem_string> string_values{};
-        //     hr = wil::reg::get_value_multistring_nothrow(key, L"multi_string_value_name", string_values);
-        //
-        // Values can be written directly from a vector of bytes - the registry type must be specified; e.g.:
-        //     wil::unique_cotaskmem_array_ptr<BYTE> raw_value{};
-        //     hr = wil::reg::get_value_binary_nothrow(key, L"binary_value_name", REG_BINARY, raw_value);
-        //
-        // Reading REG_SZ and REG_EXPAND_SZ types are done through the below templated get_value_string_nothrow and get_value_expanded_string_nothrow functions
-        // Where the template type is the type to receive the string value
-        // The default template type is std::wstring, available if the caller has included the STL <string> header
-        //
-        // Example storing a string in a wil::unique_bstr, wil::shared_bstr, wil::unique_cotaskmem_string, or wil::shared_cotaskmem_string
-        /// - These string types are passed by reference, not by pointer, because the wil types overload the & operator
-        //
-        //     wil::unique_bstr bstr_value{};
-        //     hr = wil::reg::get_value_nothrow(key, L"string_value_name", bstr_value);
-        //     // or can specify explicitly reading a string into a wil::unique_bstr type
-        //     hr = wil::reg::get_value_string_nothrow(key, L"string_value_name", bstr_value);
-        //
-        //     wil::shared_bstr shared_bstr_value{};
-        //     hr = wil::reg::get_value_nothrow(key, L"string_value_name", shared_bstr_value);
-        //     // or can specify explicitly reading a string into a wil::shared_bstr type
-        //     hr = wil::reg::get_value_string_nothrow(key, L"string_value_name", shared_bstr_value);
-        //
-        //     wil::unique_cotaskmem_string string_value{};
-        //     hr = wil::reg::get_value_nothrow(key, L"string_value_name", string_value);
-        //     // or can specify explicitly reading a string into a wil::unique_cotaskmem_string type
-        //     hr = wil::reg::get_value_string_nothrow(key, L"string_value_name", string_value);
-        //
-        //     wil::shared_cotaskmem_string shared_string_value{};
-        //     hr = wil::reg::get_value_nothrow(key, L"string_value_name", shared_string_value);
-        //     // or can specify explicitly reading a string into a wil::shared_cotaskmem_string type
-        //     hr = wil::reg::get_value_string_nothrow(key, L"string_value_name", shared_string_value);
-        //
-
-        /**
-         * \brief Reads a value under a specified key, the registry type based off the templated type passed as data
-    
```

Comment:
For some reason `clang-format` dislikes this style of doxygen comments that use backslash :/

Reply:
Using `@` syntax doesn't have this problem, but has a different issue: lines don't reflow at all

Reply:
I think it might be best to
1. Use `@` consistently instead of `\` (someone let me know if they are not interchangeable)
2. Manually format the doxygen code blocks